src/test/java/com/academy/ui/profilePage/EditClubCardWithManagerTest.java

```diff
@@ -350,4 +351,25 @@ public void checkStepThreeEditDescriptionTextArea() {
 
         softAssert.assertAll();
     }
+
+    @Test
```

Comment:
I would recommend adding a description for more information about the test

Reply:
I added @Issue below